website_project/django_project/views.py

```diff
@@ -0,0 +1,13 @@
+from django.views.generic import TemplateView
+
+
+class HomePage(TemplateView):
+    template_name = 'index.html'
+
+
+class TestPage(TemplateView):
+    template_name = 'test.html'
```

Comment:
Should we name this SuccessPage or something more official than Test? Same for any references to this view.

Reply:
Ideally the user will be redirected to the home page instead of a success page. This is just a placeholder to show that it is working currently.